### ISSUE TITLE
Open Seadragon keyboard navigation doesn't work in full-screen view

### ISSUE THREAD

**Issue by seanaery** [duke-libraries/ddr-public#409]
Noted by Mike A. in 6/2/2016 testing. The keyboard nav options that work in regular view ( https://openseadragon.github.io/examples/ui-keyboard-navigation/ ) don't do anything in full-screen view.
